### PR TITLE
Instrument Kind Verification

### LINKED ISSUE #37: Make an enum in the backend for InstrumentType
Per our demo in class today, we’ve discovered that not having the database act as a source-of-truth for available instrument kinds can be a bit of a problem, especially in our lovely strictly typed Swift app. 

This should be pretty simple; we actually did something like this for `AssetCondition` under `Assets` already. For a fairly complete list of Instruments, both frontends should have them already somewhere. In the iOS app I’m pretty sure it’s under `Instrument.swift`.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

### PR DESCRIPTION
**Pull request by charlesjhill**
Closes #37 

This pull involves adding a enum of possible Instrument kinds to the django backend and enforcing these kinds across our frontends to ensure instruments added through one UI are able to be rendered properly in other UIs. 

A breaking change was made on the backend with a couple database migrations needing made. The database might need to be re-created and its data re-seeded in order to continue testing.

### CODE REVIEW

File: DjangoBackend/ucbearcatbandsproject/bands/models.py [hunk 1]
```diff
@@ -165,7 +165,60 @@ class Instrument(Asset):
...
+                          (mellophone, 'MELLOPHONE'),
+                          (oboe, 'OBOE'), 
+                          (piano, 'PIANO'),
+                          (piccolo, 'PICCOLO'),
+                          (sopranoSaxophone, 'SOPRANOSAXOPHONE'),
+                          (sousaphone, 'SOUSAPHONE'),
+                          (tenorSaxophone, 'TENORSAXOPHONE'),
+                          (trombone, 'TROMBONE'),
+                          (trumpet, 'TRUMPET'),
+                          (tuba, 'TUBA'))
```
Comment (charlesjhill) [inline DjangoBackend/ucbearcatbandsproject/bands/models.py]: It's worth pointing out that, and we do this pretty much everywhere, that we have the order of these flipped; The constants should be the database-stored values (e.g. "TUBA", "EFLATCLARINET", etc.) and the latter values should be the human-readable form (e.g. "Tuba", "Eb Clarinet"). See [here](https://docs.djangoproject.com/en/2.2/ref/models/fields/#choices)

**Review by charlesjhill — COMMENTED** [review]
Looks fine, just need to flip the order of our enumerable and make the updates the UIs to correctly render these things

File: DjangoBackend/ucbearcatbandsproject/bands/models.py (hunk 1 above)
Comment (benhollar) [reply DjangoBackend/ucbearcatbandsproject/bands/models.py]: This should be resolved in the most recent change.

File: AngularClient/src/app/_models/Instrument.ts [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+    'Soprano Saxophone',
+    'Sousaphone',
+    'Tenor Saxophone',
+    'Trombone',
+    'Trumpet',
+    'Tuba'
+  ];
+
+
   [x: string]: any; // added to do some hacky shit disregard this
```
Comment (benhollar) [inline AngularClient/src/app/_models/Instrument.ts:33]: lol at this comment.

**Review by benhollar — APPROVED** [review]
Looks good to me now.

**Pull request merged**